parambokeh/__init__.py

```diff
@@ -238,7 +238,7 @@ def _make_widget(self, p_name):
 
         kw = dict(value=value)
         if isinstance(p_obj, param.Action):
-            def action_cb(button):
+            def action_cb():
```

Comment:
Did this change recently or was it just always broken?

Reply:
paramnb and parambokeh should definitely be consistent about whether they pass the button instance or not...

Reply:
> Did this change recently or was it just always broken?

Change where? In bokeh? I don't know for sure, but as far as I can see the on_click handler is documented as taking one argument, but it didn't seem to be working. I will double check that now. 

> paramnb and parambokeh should definitely be consistent about whether they pass the button instance or not...

I have no idea what you're talking about - could you explain?

Reply:
> on_click handler is documented as taking one argument

I retract that, I can't see documentation for it. 

Anyway, based on experiment: it takes one argument in 0.12.6 (`new`) and none in 0.12.9. I guess I will look at the source code...

What about I make it `*args,**kw` for now for cross-version compatibility?!

Reply:
> paramnb and parambokeh should definitely be consistent about whether they pass the button instance or not...

There has been some discussion in the past about what the most useful object to pass to the function associated with an Action is.  I believe (but am not sure) we changed how that works at some point, and I'm just saying that whatever we did there needs to be consistent between paramnb and parambokeh -- a param.Action should work the same, regardless of where it's used.  Not saying there is a problem here, just that we need to be careful...

Reply:
In my mind this line has 100% nothing to do with param, paramnb, parambokeh, or even paramtk...it's entirely about bokeh. So I am confused.

Reply:
Sounds like I'm probably the one confused; I can only see a little bit in the diff, not the whole context.